ui/pages/keychains/restore-vault.js

```diff
@@ -112,7 +116,7 @@ class RestoreVaultPage extends Component {
                 </Button>,
               ])}
             </Typography>
-            <Typography color={COLORS.TEXT_DEFAULT} margin={[0, 0, 4]}>
+            <Typography color={COLORS.TEXT_DEFAULT} margin={0} marginBottom={4}>
               {t('resetWalletWarning')}
             </Typography>
             <CreateNewVault
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props